test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -451,7 +451,7 @@ def _run_dependency_check_test(image, ec2_connection):
             "0.4": ["cpu", "gpu"],
             "0.5": ["cpu", "gpu"]
         },
-        "huggingface_pytorch_trcomp": {"1.9": ["gpu"]},
+        "huggingface_pytorch_trcomp": {"1.9": ["gpu"], "1.11": ["gpu"]},
```

Comment:
Currently installed version of openssl in the image is `openssl/now 1.1.1f-1ubuntu2.16 amd64` which is free from vulnerability.